UnitTests/Utilities/GeometryModelRenderer/src/PointCloudGeometryModelRenderer.h

```diff
@@ -0,0 +1,71 @@
+/*
+	Copyright (c) 2021 Technical University of Munich
+	Chair of Computational Modeling and Simulation.
+
+	TUM Open Infra Platform is free software; you can redistribute it and/or modify
+	it under the terms of the GNU General Public License Version 3
+	as published by the Free Software Foundation.
+
+	TUM Open Infra Platform is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+#ifndef OFFGEOMETRYMODELRENDERER_H
+#define OFFGEOMETRYMODELRENDERER_H
```

Comment:
This should be renamed to POINTCLOUDGEOMETRYMODELRENDERER_H

Reply:
Resolved: ec92880